shell/platform/tizen/tizen_renderer_ecore_wl2.cc

```diff
@@ -296,7 +423,17 @@ bool TizenRendererEcoreWl2::SetupEcoreWlWindow(int32_t width, int32_t height) {
 
   ecore_wl2_window_ =
       ecore_wl2_window_new(ecore_wl2_display_, nullptr, x, y, width, height);
-  ecore_wl2_window_type_set(ecore_wl2_window_, ECORE_WL2_WINDOW_TYPE_TOPLEVEL);
+  ecore_wl2_window_type_set(ecore_wl2_window_,
+                            top_ ? ECORE_WL2_WINDOW_TYPE_NOTIFICATION
+                                 : ECORE_WL2_WINDOW_TYPE_TOPLEVEL);
```

Comment:
Does EFL have any documentation for this enum? Do you know what the meaning of `ECORE_WL2_WINDOW_TYPE_NOTIFICATION` is? Shouldn't we use `ECORE_WL2_WINDOW_TYPE_NONE` instead of `ECORE_WL2_WINDOW_TYPE_TOPLEVEL` for normal windows?

Reply:
I can't find any documents for this enum, but I found [this](https://review.tizen.org/gerrit/gitweb?p=platform/upstream/efl.git;a=blob;f=src/lib/elementary/efl_ui_win.c;h=2e91df3fd24233cce7bbd501aabb38b1b3681cb5;hb=refs/heads/tizen#l4048)

I guess the notification is what we are familiar with. The document says as follows.
```
A notification window, like a warning about battery life or a new E-Mail received.
```


ECORE_WL2_WINDOW_TYPE_NONE has the same meaning as ELM_WIN_UNKNOWN.

Reply:
It's still not clear. What's different between `ECORE_WL2_WINDOW_TYPE_NOTIFICATION` and `ECORE_WL2_WINDOW_TYPE_TOPLEVEL`? The default value of `top_level` is true (https://github.com/flutter-tizen/flutter-tizen/pull/252), so the window type of normal apps will default to `ECORE_WL2_WINDOW_TYPE_NOTIFICATION` after this change. Is this intentional?